src/components/BounceRulesContainer/index.test.jsx

```diff
@@ -3,122 +3,115 @@ import renderer from "react-test-renderer";
-  createRuleModal,
-  confirmModal,
-} = Selectors;
-
-const testRules = [
-  {
-    id: 504,
-    response_code: 550,
-    enhanced_code: "",
-    regex: "no MX record for domain",
-    priority: 0,
-    description:
-      "bWFpbmx5IGxpYmVydHkgZG9tYWluIGJsb2NrIHNlZWluZyB+NTAlIG9mIGFkZHJlc3NlcyBlbmdhZ2luZyBTRyB3aWRl",
-    bounce_action: "no_action",
-  },
-];
-
-const testActivity = [
-  {
-    id: 173,
-    response_code: 421,
-    enhanced_code: "",
-    regex: "",
-    priority: 0,
-    description: "RFC5321Servicenotavailable",
-    bounce_action: "retry",
-    user_id: 1,
-    created_at: 1,
-    comment: "Inital Setup",
-    operation: "New",
-  },
-];
-
-const newRule = {
-  id: 504,
-  response_code: 551,
-  enhanced_code: "",
-  regex: "no MX record for domain",
-  priority: 0,
-  description: "new rule",
-  bounce_action: "no_action",
-};
-
-const defaultProps = {
-  rules: testRules,
-  filteredRules: testRules,
-  filterOptions: [],
-  pageIndex: 1,
-  pagesToDisplay: 5,
-  activityLog: testActivity,
-  filteredActivityLog: testActivity,
-  isFetching: false,
-  isBounceRulesTab: true,
-};
-
-const wrapper = shallow(<BounceRuleContainer {...defaultProps} />);
-
-it("should render correctly", () => {
-  const tree = renderer
-    .create(
-      <Router>
-        <BounceRuleContainer {...defaultProps} />
-      </Router>
-    )
-    .toJSON();
-  expect(tree).toMatchSnapshot();
-});
-
-it("should render a create a bounce rule", () => {
-  expect(wrapper.find(createRuleButton)).toHaveLength(1);
-});
-
-it("should render a export to csv button", () => {
-  expect(wrapper.find(csvButton)).toHaveLength(1);
-});
-
-it("should render a filter component", () => {
-  expect(wrapper.find(ruleFilter)).toHaveLength(1);
-});
-
-it("should render a rule list component", () => {
-  expect(wrapper.find(ruleTable)).toHaveLength(1);
-});
-
-it("should render paginiation", () => {
-  expect(wrapper.find(Pagination)).toHaveLength(1);
-});
+  it("should render correctly", () => {
+    const tree = renderer
+      .create(
+        <Router>
+          <BounceRuleContainer {...props} />
+        </Router>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
 
-it("should render warning when no rules available", () => {
-  wrapper.setProps({ filteredRules: [], rules: [] });
-  expect(wrapper.find(emptyRulesWarning)).toHaveLength(1);
-});
+  describe("When a user visits the bounce rule page", () => {
+    it("should render a page tab", () => {
+      expect(BounceRulesPage().find(".page-tab")).toHaveLength(1);
+    });
+
+    it("should render a create a bounce rule", () => {
+      expect(
+        BounceRulesPage()
+          .find(createRuleButton)
+          .exists()
+      ).toBeTruthy();
+    });
+
+    it("should render a export to csv button", () => {
+      expect(
+        BounceRulesPage()
+          .find(csvButton)
+          .exists()
+      ).toBeTruthy();
+    });
+
+    it("should render filter component", () => {
```

Comment:
for a lot of these render X healthcheck tests, we often merge them into one. No need to change in this PR, but something to think about for other cases if a lot of the tests are making sure the default things show up, we often combine tests to be like ('should show default state') or ('should pass healthchecks') etc. so we don't have a lot of render * tests